src/widgets/vitals/vitals-header/vital-header-title.component.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import styles from "./vital-header-title.component.scss";
+import {
+  WarningFilled20,
+  ChevronDown20,
+  ChevronUp20
+} from "@carbon/icons-react";
+import dayjs from "dayjs";
+import isToday from "dayjs/plugin/isToday";
+import { Button } from "carbon-components-react";
+import { PatientVitals } from "../vitals-card.resource";
+import { isEmpty } from "lodash-es";
+dayjs.extend(isToday);
+
+interface VitalsHeaderStateTitleProps {
+  view: string;
+  vitals: PatientVitals;
+  toggleView: Function;
```

Comment:
Please don't use `Function`. Type the function instead (e.g., `toggleView(): void`).